src/algebra/category/Group/basic.lean

```diff
@@ -42,6 +42,12 @@ attribute [to_additive] Group.has_coe_to_sort Group.large_category Group.concret
 /-- Construct a bundled `AddGroup` from the underlying type and typeclass. -/
 add_decl_doc AddGroup.of
 
+/-- Construct a bundled hom from the underlying monoid_hom. -/
+@[to_additive] def of_hom {X Y : Type u} [group X] [group Y] (f : X →* Y) : of X ⟶ of Y := f
+
+/-- Construct a bundled hom from the underlying monoid_hom. -/
+add_decl_doc AddGroup.of_hom
```

Comment:
Here and below, you can put the desired docstring in the arguments to `to_additive`:
```suggestion
@[to_additive "Construct a bundled hom from the underlying monoid_hom."]
def of_hom {X Y : Type u} [group X] [group Y] (f : X →* Y) : of X ⟶ of Y := f
```

Though it looks like there were already a bunch of other `add_decl_doc`s in the file which could also be combined with `to_additive`...

Reply:
I think sticking with the styles already in this file would be better since fixing the other stuff is probably beyond the scope of this PR.